### ISSUE TITLE
about dataset preprocess

### ISSUE THREAD

**Issue by GaoHuaZhang** [yykzjh/3D-tooth-segmentation#1]
Hello, could you please tell me how you deal with the dataset? I think the shape of your resieze in my_dataset.py is [192jue 192j72]. But in load_nrrd.py, it is [192, 192, 96]. I don't know which method you are used. In addition, I imported my own annotation file and did not seem to recognize the tag value in options. Is it because of my tagging problem? I  used 3Dslicer   before.thanks~

**Comment by yykzjh** [comment]
Hello, I am glad to answer your questions. I don't use the resize function in my_dataset.py, now I preprocess the shape to [192, 192, 96] and save them to .npy format. Your image size is determined by your GPU's memory. I also used 3D Slicer for tagging. However, I discovered that the label value exported by 3D Slicer was random, so I gave a specific class name to the segment of each category when tagging, then searched the randomly generated label value through the class name in the options when processing the annotation file, and modified it to the label value set by myself. The specific class names is in "./dataset/index_to_class.json". The function load_label_nrrd() use this file.
